storage/redis/peer_store.go

```diff
@@ -2,25 +2,25 @@
 // BitTorrent tracker keeping peer data in redis with hash.
 // There two categories of hash:
 //
-// - IPv{4,6}_{L,S}_infohash
-//	To save peers that hold the infohash, used for fast searching,
```

Comment:
Oh interesting, you (or some tool?) converted tabs to spaces in godoc. I just checked and [this official-looking example](https://tip.golang.org/doc/comment) also uses spaces, so I'm good with that!